navigation.yaml

```diff
@@ -76,7 +76,7 @@ appliance:
           path: /appliance/openhab/intel-nuc
         - title: VM
           path: /appliance/openhab/vm
-    - title: NexCloud
+    - title: NextCloud
```

Comment:
I think the C is not capital.

(see: https://nextcloud.com/ )

Reply:
You are the most correct person i know 👍. Done